Documentation/security/policy/language.rst

```diff
@@ -827,10 +827,8 @@ latter rule will have no effect.
 .. note:: Layer 7 rules are not currently supported in `HostPolicies`, i.e.,
           policies that use :ref:`NodeSelector`.
 
-.. note:: Layer 7 policies will proxy traffic through an Envoy instance provided 
-          in each Cilium agent pod. As a result, L7 traffic targeted by policies 
-          depend on the availability of the Cilium agent pod. This includes L7 
-          policies (:ref:`proxy_visibility`).
+.. note:: Layer 7 policies will proxy traffic through a node-local :ref:`envoy` instance
```

Comment:
Two suggestions here:

1. Do you mind rephrasing this sentence to remove the parenthesis? Parenthesis are great for extra, non-essential pieces of information or side comments, and I don't think they fit here.
2. Can we keep the comment that L7 traffic targeted by policies depends on the availability of the Cilium agent pod in the case that Envoy is embedded in the agent?